FileLiberator/DownloadDecryptBook.cs

```diff
@@ -153,8 +155,8 @@ private static (string destinationDir, bool movedAudioFile) MoveFilesToBooksDir(
             foreach (var f in sortedFiles)
             {
                 var dest
-                    = AudibleFileStorage.Audio.IsFileTypeMatch(f)
```

Comment:
IsFileTypeMatch should probably still be used here

Reply:
Nice one! Have used that instead.